src/components/BookController.jsx

```diff
@@ -0,0 +1,9 @@
+const BookController = () => {
+  return (
+    <>
+      <div>Book Controllers here</div>
+    </>
+  );
+};
+
+export default BookController;
```

Comment:
good job by creating a separate component for the book 👍🏼  , but it needs to 
Take the book data as a prop and renders the title and the author, and havea button for removing books
the button wouldn't remove the books yet, this is just a placeholder for now